node/pkg/governor/governor_monitoring.go

```diff
@@ -96,14 +95,18 @@ import (
 )
 
 // Admin command to display status to the log.
-func (gov *ChainGovernor) Status() string {
+func (gov *ChainGovernor) Status() (resp string) {
 	gov.mutex.Lock()
 	defer gov.mutex.Unlock()
 
 	startTime := time.Now().Add(-time.Minute * time.Duration(gov.dayLengthInMinutes))
-	var resp string
 	for _, ce := range gov.chains {
-		valueTrans := sumValue(ce.transfers, startTime)
+		valueTrans, err := sumValue(ce.transfers, startTime)
+		if err != nil {
+			// Print an error to the logs. We don't want to actually return an error or otherwise stop
```

Comment:
Sorry for a very minor nit pick, but the comment is wrong. You're not printing to the logs, you're returning an error message.